temporal/t.downgrade/t.downgrade.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
Just a minor detail, UTF-8 is the default with [Python 3](https://docs.python.org/3.5/howto/unicode.html#python-s-unicode-support), no need for the coding line.

Reply:
Copied from other `t.*` modules.

Reply:
> Copied from other `t.*` modules.

I noticed they still contain that line, which is unnecessary there too. They may be removed in a spring clean for Python 3 one day, but I think there's no need to add this to a new file.

Reply:
Addressed in #1461